Simplified/Reader2/ReaderStackConfiguration/LibraryService.swift

```diff
@@ -15,6 +15,43 @@ import UIKit
 import R2Shared
 import R2Streamer
 
+// =============================================================================
+// MARK: - Workaround for VoiceOver Issues
+
+/// Temporary workaround to continue using the old Readium `PublicationServer`.
+///
+/// Readium 2.5.0 deprecates `PublicationServer` and introduces
+/// `GCDHTTPServer` in its place. However, this causes significant scrolling
+/// and book content navigation issues with VoiceOver. By casting your
+/// `PublicationServer` references to `PublicationServing`, this workaround
+/// silences the deprecation warnings until we find an official solution.
+protocol PublicationServing: ResourcesServer {
+  func add(_ publication: Publication) throws
+  func removeAll()
+}
+
+extension PublicationServer: PublicationServing {
+  @available(*, deprecated, message: "To suppress this warning, cast to PublicationServing protocol")
+  func add(_ publication: Publication) throws {
+    try add(publication, at: UUID().uuidString)
```

Comment:
the UUID string was a default param for the `PublicationServer::add(_, at:)` method, but we can't have default params in protocols so I have to add it here explicitly.